src/__tests__/dynamicModuleHelper.spec.ts

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright 2021 Byndyusoft
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint-disable max-classes-per-file */
+
+import { DynamicModuleHelper } from "~/src";
+
+const CONFIG_TOKEN = Symbol("CONFIG_TOKEN");
+
+class TestModule {}
+
+class TestImportedModule {}
+
+class TestProvider {}
+
+describe("DynamicModuleHelper", () => {
+  describe(".register", () => {
+    it("must add config to providers if providers exists", () => {
```

Comment:
What if use [betterspecs guideline](https://github.com/Byndyusoft/knowledge-base/blob/master/testing-guidelines.md)?

Reply:
Discussion moved to https://github.com/Byndyusoft/nest-dynamic-module/issues/21